src/js/tech/html5.js

```diff
@@ -794,6 +794,40 @@ class Html5 extends Tech {
       }
     }
   }
+
+  /**
+   * Gets available media playback quality metrics as specified by the W3C's Media
+   * Playback Quality API.
+   *
+   * @see https://wicg.github.io/media-playback-quality/
+   *
+   * @return {Object}
+   *         An object with supported media playback quality metrics
+   */
+  getVideoPlaybackQuality() {
+    if (typeof this.el().getVideoPlaybackQuality === 'function') {
+      return this.el().getVideoPlaybackQuality();
+    }
+
+    const videoPlaybackQuality = {};
+
+    if (typeof this.el().webkitDroppedFrameCount !== 'undefined' &&
```

Comment:
Judging by https://bugs.webkit.org/attachment.cgi?id=80515&action=diff (which I think is the right patch), the webkit prefixed properties were added at the same time.

As for the mozilla prefixed properties, there do seem to be some:`mozParsedFrames`, `mozDecodedFrames`, `mozPresentedFrames`, `mozPaintedFrames`, and `mozFrameDelay`, but since they aren't 1:1 matches, I'm not sure it would be worth trying to use them.